### PR TITLE
[DOCS] Added descriptions to Camera2D

### PR DESCRIPTION
**Pull request by sednave**
Added descriptions to some member variables.

### CODE REVIEW

File: doc/classes/Camera2D.xml [hunk 1]
```diff
@@ -333,6 +333,7 @@
 			Bottom margin needed to drag the camera. A value of [code]1[/code] makes the camera move only when reaching the edge of the screen.
 		</member>
 		<member name="drag_margin_h_enabled" type="bool" setter="set_h_drag_enabled" getter="is_h_drag_enabled">
+			If [code]true[/code] this camera only moves when reaching the horizontal drag margins. If [code]false[/code] the camera moves horizontally regardless of margins.
```
Comment (cbscribe) [inline doc/classes/Camera2D.xml]: Please add default value

File: doc/classes/Camera2D.xml [hunk 2]
```diff
@@ -361,6 +363,7 @@
 			Right scroll limit in pixels. The camera stops moving when reaching this value.
 		</member>
 		<member name="limit_smoothed" type="bool" setter="set_limit_smoothing_enabled" getter="is_limit_smoothing_enabled">
+			If [code]true[/code] smooth camera when reaching camera limits.
```
Comment (cbscribe) [inline doc/classes/Camera2D.xml]: This feels incomplete. I think it needs an explanation of what "smooth camera" means.

File: doc/classes/Camera2D.xml [hunk 3]
```diff
@@ -371,8 +374,10 @@
 		<member name="rotating" type="bool" setter="set_rotating" getter="is_rotating">
 		</member>
 		<member name="smoothing_enabled" type="bool" setter="set_enable_follow_smoothing" getter="is_follow_smoothing_enabled">
+			If [code]true[/code] this camera moves with smoothing.
```
Comment (cbscribe) [inline doc/classes/Camera2D.xml]: As above, describe what smoothing does.

File: doc/classes/Camera2D.xml [hunk 4]
```diff
@@ -371,8 +374,10 @@
...
 		</member>
 		<member name="smoothing_speed" type="float" setter="set_follow_smoothing" getter="get_follow_smoothing">
+			Speed at which the camera moves to its current smoothing destination.
```
Comment (cbscribe) [inline doc/classes/Camera2D.xml]: The camera's speed when moving to its...

**Review by cbscribe — CHANGES_REQUESTED** [review]

File: doc/classes/Camera2D.xml [hunk 5]
```diff
@@ -369,10 +371,13 @@
 			The camera's offset, useful for looking around or camera shake animations.
 		</member>
 		<member name="rotating" type="bool" setter="set_rotating" getter="is_rotating">
+			If [code]true[/code] the camera can rotate during gameplay. Default value: [code]false[/code].
 		</member>
 		<member name="smoothing_enabled" type="bool" setter="set_enable_follow_smoothing" getter="is_follow_smoothing_enabled">
+			If [code]true[/code] the camera gradually changes its velocity vector. Default value: [code]false[/code].
```
Comment (cbscribe) [inline doc/classes/Camera2D.xml]: Hmm, this makes it sound like the camera just moves by itself.  How about:

    If [code]true[/code] the camera's velocity changes gradually, preventing sudden starts and stops.

File: doc/classes/Camera2D.xml [hunk 6]
```diff
@@ -369,10 +371,13 @@
...
 		</member>
 		<member name="smoothing_speed" type="float" setter="set_follow_smoothing" getter="get_follow_smoothing">
+			The camera's acceleration if smoothing is enabled.
```
Comment (cbscribe) [inline doc/classes/Camera2D.xml]: Not sure about this one. Is it an acceleration? The name says speed.

File: doc/classes/Camera2D.xml [hunk 7]
```diff
@@ -333,6 +333,7 @@
 			Bottom margin needed to drag the camera. A value of [code]1[/code] makes the camera move only when reaching the edge of the screen.
 		</member>
 		<member name="drag_margin_h_enabled" type="bool" setter="set_h_drag_enabled" getter="is_h_drag_enabled">
+			If [code]true[/code] this camera only moves when reaching the horizontal drag margins. If [code]false[/code] the camera moves horizontally regardless of margins. Default value: [code]true[/code].
```
Comment (cbscribe) [inline doc/classes/Camera2D.xml]: Everywhere else you say "the camera" instead of "this camera". You should make it consistent.

**Review by cbscribe — APPROVED** [review]

**Comment by cbscribe** [comment]
Looks good, please squash the commits.

**Comment by akien-mga** [comment]
Squashed and merged manually as 8f382879c.

**Pull request closed without merging**